src/binary/requests/state.ts

```diff
@@ -0,0 +1,14 @@
+import { StateType } from "../../consts";
+import { State } from "../state";
+import { getState } from "./requests";
+import { openConfigWithSource } from "../../commandsHandler";
+
+export default async function getStateA(): Promise<State | null | undefined> {
+  const state = await getState();
+
+  if (state?.should_open_hub) {
+    void openConfigWithSource(StateType.INFO)();
```

Comment:
Is this firing a command? Looks weird

Reply:
it opens the hub... im not sure what you mean

Reply:
why is it here?

Reply:
where else should it be?